frontend/templates/views/partials/footer.j2

```diff
@@ -60,7 +60,35 @@
       </ul>
       <div class="ap-o-footer-copyright">&copy; 2019 AMP Open Source Project</div>
     </div>
+
   </div>
+  <amp-position-observer 
```

Comment:
To be honest I am not a fan of introducing another instance of `amp-position-observer` and `amp-animation` for this, I think I just dealt with this on another branch by fiddling around with the z-indexes and making the sidebar toggle vanish behind the footer. What do you think?